explainer.md

```diff
@@ -0,0 +1,147 @@
+# Open Screen Protocol Explainer
+
+The Open Screen Protocol is a standard way for two agents to discover each other
+on the local network, create a secure network connection, and allow an
+application on one agent to render media on the other.
+
+This capability allows users to discover media on one device (such as a laptop
+or smartphone) and play it on another device (such as a smart TV, set top box,
+or connected speaker).   We call this scenario "second screen" functionality.
+
+## Why should users care?
+
+Currently there are many solutions that allow media accessed on one device to be
+played on a different device.  Examples include [Google
+Cast](https://developers.google.com/cast/), [Apple
+AirPlay](https://www.apple.com/airplay/), and [Spotify
+Connect](https://www.spotify.com/us/connect/).
+
+However, the application, browser and/or OS largely determine what devices work
+together for media playback.  Users can't be easily guaranteed that the content
+that they watch and the browser they choose will be able to interface with the
+smart TV or speaker that they have or want.
+
+## Why should Web developers care?
+
+Web developers who want to integrate second screen functionality into their
+applications need to write different code for each browser to interface with the
+devices supported by that browser.  This discourages wide adoption of remote
+playback functionality.  For applications that do add second screen support,
+they tend to pick one device type versus integrating multiple device-specific
+SDKs into their application.
+
+## Assumptions and Goals
+
+The primary goal of the Open Screen Protocol is to provide a complete network
+protocol to allow devices from different vendors to implement second screen
+APIs and work together. 
+
+The protocol specifically targets the [requirements of second screen Web
+APIs](requirements.md): the [Remote Playback
+API](https://w3c.github.io/remote-playback/) and the [Presentation
+API](https://w3c.github.io/presentation-api/).
+
+A design goal is to re-use existing network and data standards like mDNS, QUIC
+and CBOR as building blocks, versus inventing our own.
+
+The protocol also provides [security and privacy
+guarantees](requirements.md#privacy-and-security), especially with regards to
+other devices on the network.
+
+We assume that the protocol should work on devices with [low-end CPUs and
+constrained memory](device_specs.md), which are typical in special purpose
+devices like smart TVs and connected speakers.  We also assume the protocol will
+be used on [battery-powered devices](requirements.md#hardware-and-efficiency). 
+
+## Non-Goals
+
+A non-goal is to enable every Web application to be able to successfully render
```

Comment:
Any other non goals?

Reply:
Well the list of "non-goals" could be boundless :-)  We can add to this section based on feedback.

I added some language under security and privacy, since there the security aspects of the protocol are intentionally limited in scope.